### DISCUSSION TITLE
#22. VATZ biweekly meeting at 2022-11-23 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#299] [General]
### 1. Overall
<img width="1207" alt="image" src="https://user-images.githubusercontent.com/6308023/200979326-09bf3ec2-1eb6-4ea5-95d1-a3daccd02eef.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
20 | 8/22 | 36
21 | 1/14| 7

---
     
# Discussion
- CI
- Exporter
- Docker

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/308
- https://github.com/dsrvlabs/vatz/issues/238

**Comment by rootwarp** [comment]
# Joonkyo

- #298 
- https://github.com/dsrvlabs/vatz/pull/307

**Comment by heejin-github** [comment]
## 희진
- #304 
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/23
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/24
- #305
  - https://github.com/dsrvlabs/vatz-plugin-sysutil/pull/6

**Comment by skonhwang** [comment]
# 승곤
- no issue

**Comment by kim201212** [comment]
# 형기

- #196 
   - #311

**Comment by gnongs** [comment]
# 진홍
- no issue

**Comment by meetrick** [comment]
# 황제
- https://github.com/dsrvlabs/vatz/issues/303
